resources/views/posts/create.blade.php

```diff
@@ -50,10 +50,23 @@
                         <div class="text-danger mt-1 mb-1">*{{ $message }}*</div>
                     @enderror
                 </div>
-            </div>
-            <div class="col-xs-12 col-sm-12 col-md-12 text-center">
-                <button type="submit" class="btn btn-primary mt-3">Submit</button>
-            </div>
-        </div>
+                <h3 class="text-primary mt-3">Choose Category</h3>
+                <div>
+                    <select class="form-select mt-3" multiple aria-label="size 3" name="category[]" multiple>
+                        <option></option>
+                        @foreach ($categories as $category)
+                            @if ($loop->index === 0)
```

Comment:
if `@foreach ($categories as $index => $category)`, 
you can use `@if ($index === 0)`